test/saml20.tests.js

```diff
@@ -23,6 +24,38 @@ describe('saml 2.0 assertion', function () {
     nameIdentifierFormat: 'urn:oasis:names:tc:SAML:1.1:nameid-format:unspecified'
   };
 
+  it('should reject expired cert embedded in saml assertion', function(done){
+    // Turns out the cert we generate expired in 2013
```

Comment:
Maybe rephrase or remove so maintainers don't start questioning what this is/was.